src/main/java/com/stefanovskyi/gitdeceiver/GitAuthData.java

```diff
@@ -35,4 +37,8 @@ public String getPassword() {
         return password;
     }
 
+    public int getAmountOfDays() {
+        return amountOfDays;
```

Comment:
The `amountOfDays` is not a part of `AuthData`. Please remove it out of here.

Reply:
Done